src/Deserializers/ClaimDeserializer.php

```diff
@@ -69,9 +69,9 @@ private function hasCorrectType( $serialization ) {
 	/**
 	 * @see Deserializer::deserialize
 	 *
-	 * @param mixed $serialization
+	 * @param array $serialization
 	 *
-	 * @return object
+	 * @return Claim|Statement
```

Comment:
Uh, didn't see that. `Claim` should last here because `Statement` is still a subclass of `Claim` in the version of DM we depend here.